thread.c

```diff
@@ -1771,7 +1771,7 @@ VALUE
 rb_thread_io_blocking_call(rb_blocking_function_t *func, void *data1, int fd, int events)
 {
     rb_execution_context_t *volatile ec = GET_EC();
-    rb_thread_t *th = rb_ec_thread_ptr(ec);
+    rb_thread_t * volatile th = rb_ec_thread_ptr(ec);
```

Comment:
```suggestion
    rb_thread_t *volatile th = rb_ec_thread_ptr(ec);
```

sorry to be annoying, but there is no space between the two on just the line above 😆